### PR TITLE
Add support for textareas - closes #179

### LINKED ISSUE #179: Experienced issue using textareas
Have you encountered any issues when using medium-editor with textareas? I had no trouble loading up this plugin when using a div, but it would not load if the editor is based on a textarea.

### PR DESCRIPTION
**Pull request by orthes**
Fix #179 

- Update dependencies
- Support for textareas. If the plugin is initialized on a textarea, the plugin finds the associated div and it initializes itself on this div instead.

Side note: ```src/js/templates.js``` is different because I updated ```grunt-contrib-handlebars``` version.

### CODE REVIEW

**Comment by j0k3r** [comment]
Looks good to me.
Do you need to add anything else?

**Comment by orthes** [comment]
Nope, we can merge it. I will make another PR with support for ```editableInput``` event.

File: bower.json [hunk 1]
```diff
@@ -22,9 +22,9 @@
   ],
   "dependencies": {
     "jquery": ">=1.9.0",
-    "medium-editor": "^4.12.0",
+    "medium-editor": "^5.7.0",
```
Comment (j0k3r) [inline bower.json]: Finally :smile:

**Comment by unkunus** [comment]
working perfectly in some tests. thx

**Comment by unkunus** [comment]
there is still a problem when the editor is disabled with editor.destroy() and then enabled again with editor.setup()

medium-editor-insert-plugin.js:302 Uncaught TypeError: Cannot read property 'enable' of undefined

**Comment by j0k3r** [comment]
Could you open a separate issue about that ?
And if you could provide a jsfiddle to reproduce the bug, it'll perfect.

**Comment by unkunus** [comment]
ok, set up a jsfiddle and opened an issue. thanks and greetings,
frank

https://jsfiddle.net/ho4xtnfc/12/



Am 10.09.2015 um 21:01 schrieb Jeremy Benoist:
> Could you open a separate issue about that ?
> And if you could provide a jsfiddle to reproduce the bug, it'll perfect.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/orthes/medium-editor-insert-plugin/pull/215#issuecomment-139345905>.
>

-- 
Frank Amos, Mühlrain 64,  70180 Stuttgart

**Pull request merged**